src/test/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsTest.java

```diff
@@ -26,4 +29,40 @@ void testGetReferenceTitle(){
         assertEquals("Reference Number", result);
     }
 
+    @Test
+    void testGetLoginEmailSuccessful() {
+        String email = "test@gmail.com";
+        Map<String, Object> userProfile = Map.of("email", email);
+        Map<String, Object> signInInfo = Map.of("user_profile", userProfile);
+        SessionService sessionService = new SessionService() {
```

Comment:
This can be replaced with a lambda: 
`SessionService sessionService = () -> Map.of("signin_info", signInInfo);`
The IDE suggested this for me, maybe after the break let's have a look at the settings on your IntelliJ.